atomistics/workflows/__init__.py

```
@@ -26,10 +28,13 @@
 try:  # in case phonopy is not installed
     from atomistics.workflows.phonons.workflow import PhonopyWorkflow
     from atomistics.workflows.quasiharmonic import QuasiHarmonicWorkflow
-
+except ImportError as e:
+    warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)